mirrord/config/src/config.rs

```diff
@@ -15,6 +15,12 @@ pub enum ConfigError {
     #[error("value for {1:?} not provided in {0:?} (env override {2:?})")]
     ValueNotProvided(&'static str, &'static str, Option<&'static str>),
 
+    #[error("value {0:?} for {1:?} is invalid.")]
+    InvalidValue(String, &'static str),
+
+    #[error("default value {0:?}")]
+    InvalidDefaultValue(&'static str),
```

Comment:
How can we have an invalid default value? Is this an user provided default?

Reply:
Can't we move this to the macro crate and have it checked at compile time? So we get a nice error if we put a bonkers value as the default somewhere?